Client/Frontend/Strings.swift

```diff
@@ -69,24 +63,6 @@ extension String {
 extension String {
     public struct Bookmarks {
 
-        public struct Actions {
-            public static let Add = MZLocalizedString(
-                "Bookmarks.Actions.Add",
-                value: "Add",
-                comment: "A label indicating the action of adding a web page as a bookmark.",
-                lastUpdated: .v96)
-            public static let BookmarkAllTabs = MZLocalizedString(
-                "Bookmarks.Actions.BookmarkAllTabs",
-                value: "Bookmark All Tabs",
-                comment: "A label indicating the action of bookmarking all currently open non private tabs.",
-                lastUpdated: .v96)
-            public static let BookmarkCurrentTab = MZLocalizedString(
-                "Bookmarks.Actions.BookmarkCurrentTab",
-                value: "Bookmark Current Tab",
-                comment: "A label indicating the action of bookmarking the current tab.",
-                lastUpdated: .v96)
-        }
```

Comment:
@dnarcese do we plan on working on those bookmarks changes? Can those be removed?

Reply:
Lets remove them